Dependencies/xr/Source/ARCore/XR.cpp

```diff
@@ -657,6 +657,27 @@ namespace xr
             return {pose, reinterpret_cast<NativeAnchorPtr>(arAnchor)};
         }
 
+        Anchor DeclareAnchor(NativeAnchorPtr anchor)
+        {
+            ArAnchor* arAnchor{reinterpret_cast<ArAnchor*>(anchor)};
+            const auto iterator = std::find(arCoreAnchors.begin(), arCoreAnchors.end(), arAnchor);
```

Comment:
Isn't the code in NativeXR.cpp already guarding against this? How can this happen?

Reply:
this is now removed